apps/transport/lib/transport_web/plugs/custom_secure_browser_headers.ex

```diff
@@ -0,0 +1,61 @@
+defmodule TransportWeb.Plugs.CustomSecureBrowserHeaders do
+  @moduledoc """
+    Call the put_secure_browser_headers Plug and add some CSP headers
+  """
+
+  def init(options), do: options
+
+  def call(conn, _opts) do
+    csp_headers = csp_headers(Application.fetch_env!(:transport, :app_env))
+    Phoenix.Controller.put_secure_browser_headers(conn, csp_headers)
+  end
+
+  @doc """
+  Returns content-security-policy headers, depending on the APP_ENV value
+
+  iex> csp_headers("")
+  %{}
+  iex> csp_headers("test")
+  %{}
+  iex> match?(%{"content-security-policy" => _csp_content}, csp_headers(:prod))
+  true
+  iex> match?(%{"content-security-policy" => _csp_content}, csp_headers(:staging))
+  true
+  """
+  def csp_headers(app_env) do
```

Comment:
On devrait pouvoir utiliser Sentry pour collecter des erreurs de CSP, voir

- https://docs.sentry.io/product/security-policy-reporting/#content-security-policy
- https://sentry.io/settings/transport-data-gouv-fr/projects/transport-site/security-headers/

Il faudra surement ajouter une variable d'environnement

Reply:
ça a l'air plus [compliqué](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Content-Security-Policy/report-to), on va s'en tenir à `report-uri` pour le moment.